build.sbt

```diff
@@ -151,6 +151,8 @@ lazy val s3 = monixConnector("s3", Dependencies.S3)
 
 lazy val gcs = monixConnector("gcs", Dependencies.GCS)
 
+lazy val es =  monixConnector("es", Dependencies.ES)
```

Comment:
we'd need to add that as a dependency to main module _monnix-connect_ in both `.depends(...)` and `aggregate(...)`